ReactWindows/ReactNative.Shared/Bridge/NativeModuleBase.cs

```diff
@@ -63,35 +62,9 @@ protected NativeModuleBase()
         /// Factory responsible for creating delegates for method invocations.
         /// </param>
         protected NativeModuleBase(IReactDelegateFactory delegateFactory)
-            : this(delegateFactory, null)
-        {
-        }
-
-        /// <summary>
-        /// Instantiates a <see cref="NativeModuleBase"/>.
-        /// </summary>
-        /// <param name="actionQueue">
-        /// The action queue that native modules should execute on.
-        /// </param>
-        protected NativeModuleBase(IActionQueue actionQueue)
```

Comment:
Now I'm concerned that we can't customize the ActionQueue for a basic native module.

Reply:
Is that so bad? They can just switch to a Context-based module. It's not ideal but this design avoids being a breaking change.

I plan to update this PR to have a unit  test as Matt suggested but I've been occupied by other priorities recently.

Reply:
Is that so bad? They can just switch to a Context-based module.

It's not ideal but I had good reasons for choosing this design. See my response to your comment "Ugh - we're going to need a 40 page...." for details.

---
In reply to: [174974769](https://github.com/Microsoft/react-native-windows/pull/1650#discussion_r174974769) [](ancestors = 174974769)